runtime/src/message_processor.rs

```diff
@@ -64,6 +64,7 @@ impl MessageProcessor {
         blockhash: Hash,
         lamports_per_signature: u64,
         current_accounts_data_len: u64,
+        consumed_compute_units: &mut Option<&mut u64>,
```

Comment:
What's the rationale for `&mut Option<>` here? It sounds like a `Option<&mut u64>` would be ok.

Reply:
`.as_deref_mut()`?